src/components/themed/ManageTokensRow.tsx

```diff
@@ -84,9 +85,9 @@ export const ManageTokensRowComponent = (props: Props) => {
         <EdgeText style={styles.displayName}>{token.displayName}</EdgeText>
       </View>
       {!isCustom ? null : (
-        <TouchableOpacity style={styles.editIcon} onPress={handleEdit}>
+        <EdgeTouchableOpacity style={styles.editIcon} onPress={handleEdit}>
```

Comment:
This guy has also incorporated the `usePendingPress` hook, and uses it to drive an animation, so no need for the `EdgeTouchableOpacity`.